en/index.md

```diff
@@ -14,62 +14,60 @@ This guide contains everything you need to assemble, configure, and safely fly a
 
 :::warning
 This guide is for the _development_ version of PX4 (`main` branch).
-Use the **Version** selector for the current _stable_ version.
+Use the **Version** selector to find the current _stable_ version.
 
 Documented changes since the stable release are captured in the evolving [release note](releases/main.md).
 :::
 
 ## How Do I Get Started?
 
-[Getting Started](getting_started/index.md) should be read by all users!
+[Basic Concepts](../getting_started/px4_basic_concepts.md) should be read by all users!
 It provides an overview of PX4, including features provided by the flight stack (flight modes and safety features) and the supported hardware (flight controller, vehicles, airframes, telemetry systems, RC control systems).
 
-Depending on what you want to achieve, the following tips will help you navigate through this guide:
+If you're just getting started with flying drones then we recommend [multicopters](frames_multicopter/index.md), because they are easier to fly than planes and less complicated to build than VTOLs.
+Once you're familiar with multicopters, then moving onto planes and VTOL is much easier.

```

Comment:
Having flown a multicopter and a plane definitely makes VTOLs easier. I agree planes are harder because they cannot stop in air and the support in terms of PX4 getting started is still slightly worse probably because of also less demand but in terms of basic stability and tuning they are easier no? All perspective dependent, I wonder what @sfuhrer thinks.

Reply:
I guess it depends where you come from and what you aim for. If you have flown planes manually then yes, they're straight forward. But if you anyway want to end up operating MCs in auto modes then don't go through the struggle with planes first...

Reply:
I honestly challenge this section, there is no general advice on what to start with, MC vs FW. It's only really valid for VTOLs - I recommend that you fly a MC before a VTOL.

Reply:
I can't remember where I heard this opinion and if you both disagree with it there's no need for it - deleted. 

I suspect it is actually true that you're best off starting with MC because the documentation is much more comprehensive, and it is easier to get a developer kit etc. But we don't need to say that, and this docs effort is designed to address that problem.